fvtest/porttest/si.cpp

```diff
@@ -2703,7 +2731,58 @@ TEST(PortSysinfoTest, sysinfo_cgroup_get_memlimit)
 				outputErrorMessage(PORTTEST_ERROR_ARGS, "Expected omrsysinfo_cgroup_get_memlimit and omrsysinfo_get_physical_memory to return same value, but omrsysinfo_cgroup_get_memlimit returned %ld and omrsysinfo_get_physical_memory returned %ld\n", cgroupMemLimit, physicalMemLimit);
 			}
 		}
+
+		if (CgroupTest::isV1Available) {
+			if (OMRPORT_ERROR_SYSINFO_CGROUP_MEMLIMIT_NOT_SET == rc) {
+				EXPECT_GT(CgroupTest::memLimit, omrsysinfo_get_physical_memory()) << "Cgroup memory is set but omrsysinfo_cgroup_get_memlimit returned not set";
+			} else if (0 == rc) {
+				EXPECT_EQ(CgroupTest::memLimit, cgroupMemLimit);
+			}
+		} else {
+			if (OMRPORT_ERROR_SYSINFO_CGROUP_MEMLIMIT_NOT_SET == rc) {
```

Comment:
`CgroupTest::SetUpTestCase` will fail if `isV1Available != isV2Available`. It's implicit. Can we make a note in all such `else` blocks that it will only be invoked if `isV2Available == true`?